packages/react/src/components/password-creation-input/password-creation-input.tsx

```diff
@@ -0,0 +1,140 @@
+import { ChangeEvent, useState, VoidFunctionComponent, useMemo } from 'react';
+import styled from 'styled-components';
+import { FieldContainer } from '../field-container/field-container';
+import { IconButton } from '../buttons/icon-button';
+import { TextInput } from '../text-input/text-input';
+import { useTranslation } from '../../i18n/use-translation';
+import { getPasswordStrength } from './password-strength';
+import { PasswordStrengthMeter } from './password-strength-meter';
+import { PasswordCondition } from './password-condition';
+import { getDefaultValidationConditions, ValidationCondition } from './validation-condition';
+import { v4 as uuid } from '../../utils/uuid';
+
+const StyledFieldContainer = styled(FieldContainer)`
+    > :nth-child(2) {
+        margin-bottom: 0;
+    }
+`;
+
+const StyledUl = styled.ul`
+    font-size: 0.75rem;
+    margin: 0 0 var(--spacing-half) 0;
+    padding: 0;
+`;
+
+const PasswordContainer = styled.div`
+    display: flex;
+    margin-bottom: 12px;
```

Comment:
Pour utiliser les variables css on pourrait changer pour
```suggestion
    margin-bottom: calc(var(--spacing-1x) * 1.5);
```